commit 597565556581d59641c0be50acaae87f7391a91b
Author: Vladimir Oltean <vladimir.oltean@nxp.com>
Date:   Wed Feb 17 22:33:48 2021 +0200

    net: mscc: ocelot: select PACKING in the Kconfig
    
    Ocelot now uses include/linux/dsa/ocelot.h which makes use of
    CONFIG_PACKING to pack/unpack bits into the Injection/Extraction Frame
    Headers. So it needs to explicitly select it, otherwise there might be
    build errors due to the missing dependency.
    
    Fixes: 40d3f295b5fe ("net: mscc: ocelot: use common tag parsing code with DSA")
    Reported-by: Naresh Kamboju <naresh.kamboju@linaro.org>
    Signed-off-by: Vladimir Oltean <vladimir.oltean@nxp.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

diff --git a/drivers/net/ethernet/mscc/Kconfig b/drivers/net/ethernet/mscc/Kconfig
index ee7bb7e24e8e..c0ede0ca7115 100644
--- a/drivers/net/ethernet/mscc/Kconfig
+++ b/drivers/net/ethernet/mscc/Kconfig
@@ -14,6 +14,7 @@ if NET_VENDOR_MICROSEMI
 # Users should depend on NET_SWITCHDEV, HAS_IOMEM
 config MSCC_OCELOT_SWITCH_LIB
 	select REGMAP_MMIO
+	select PACKING
 	select PHYLIB
 	tristate
 	help